<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="IZDT-211" name="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage" tests="7" failures="1" timestamp="10 Aug 2018 05:04:52 GMT" time="29.190" errors="0">
  <testcase name="testPG_1" time="19.708" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage"/>
  <testcase name="testPG_2" time="0.882" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage"/>
  <testcase name="testPG_3" time="1.976" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage"/>
  <testcase name="testPG_4" time="0.297" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage"/>
  <testcase name="testPG_5" time="0.033" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage">
    <skipped/>
  </testcase> <!-- testPG_5 -->
  <testcase name="testPG_6" time="5.796" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage">
    <failure type="java.lang.AssertionError" message=" Responsive Content  data  passing on Responsive Editor  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Responsive Content  data  passing on Responsive Editor  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage.testPG_6(ScenarioSpecialPage.java:96)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testPG_6 -->
  <testcase name="testPG_7" time="0.498" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage"/>
</testsuite> <!-- com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage -->
